lib/Mail/DMARC/PurePerl.pm

```diff
@@ -266,7 +266,7 @@ sub is_spf_aligned {
 
     my $from_dom = $self->header_from or croak "header_from not set!";
```

Comment:
Since $from_dom is used again in a comparison later, you should probably lower case it here: 

```suggestion
    my $from_dom = lc $self->header_from or croak "header_from not set!";
```